src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -274,7 +276,14 @@ valid_file_mem_backends = @DEFVALIDFILEMEMBACKENDS@
 # The value means the number of pcie_root_port
 # This value is valid when hotplug_vfio_on_root_bus is true and machine_type is "q35"
 # Default 0
-#pcie_root_port = 2
+pcie_root_port = 2
+
```

Comment:
I have mixed feelings about changing this default value.  Having some root ports by default would be valuable for my uses cases as well, but I believe it wasn't the default for a reason - I think it slowed down guest startup for cases that didn't need it.

Reply:
It's a mistake. Recovered back.